helper_commands/management/commands/download_data.py

```diff
@@ -52,5 +53,9 @@ def handle(self, *args, **kwargs):
             settings.SYNC_FIXTURE_LOCATION,  # local temp filename
         ]  # command to pull down fixture to local file, with aws env vars
         aws_open(download_s3)
-        management.call_command('flush', interactive=False)
-        management.call_command('loaddata', settings.SYNC_FIXTURE_LOCATION)
+        table_names = run_sql(
+            "select tablename from pg_tables where schemaname = 'public'")
+        for table_name in table_names:
+            print(table_name)
+            drop_table(table_name[0])
```

Comment:
Given that this now drops tables (instead of clearing them) it would be useful to update the docstring to make it clear that it will drop all tables.